### ISSUE TITLE
Empty summary notification

### ISSUE THREAD

**Issue by CerberusCode** [8bitPit/Niagara-Issues#1681]
**Describe the bug**
I read all of my summary, but yet i still it in my homescreen notifying that i have a summary.

**To Reproduce**

**Smartphone (please complete the following information):**
 - Device: OPPO F12 Pro
 - Android version: Android 11

**Screenshots**
![Screenshot_2021-08-07-10-00-42-06_a4824db61f40c53807964ea8b14c92fe](https://user-images.githubusercontent.com/65819283/128584667-e66010b8-dd94-4da2-82a1-d53611dbf624.jpg)

**Comment by sandeshdroid** [comment]
Just click on close summary & schedule next button at bottom and you are good to go.

**Comment by CerberusCode** [comment]
So its really a bug?

**Comment by CerberusCode** [comment]
Hello, so its not really a bug?

**Comment by PemikM** [comment]
> Just click on close summary & schedule next button at bottom and you are good to go.

But it would be nice if you won't have to do this.

**Comment by Scott123456** [comment]
I agree and this has long been an annoyance for me. Would love to see it fixed (or changed) to be more user friendly. Summary notification should dismiss automatically when it's empty.

**Comment by ssssamstrup** [comment]
Seconding it would be lovely if the summary would disappear/schedule next after dismissing the last notification. 😊

**Comment by TFackler** [comment]
Sounds like a good idea. We will add this to our list of future changes 👍

**Comment by ssssamstrup** [comment]
Idea: It would be great if the screen had a small not congratulating you on going through and emptying the summary and telling you when the next summary will appear „We'll schedule your new summary in [X] hours. Hope you're having a great day!”.

Then it could either disappear after a second or two (could just be a Android toast/flash message).
If this was implemented I wouldn't mind having to press a button to dismiss. Acknowledging the end has been reached/summary cleared means it doesn't feel useless having to click the button, like it does now.
